src/OpenTelemetry.Shared/IPersistentBlob.cs

```diff
@@ -0,0 +1,48 @@
+// <copyright file="IPersistentBlob.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Shared
+{
+    /// <summary>
+    /// API to Read, Write and Delete a blob.
+    /// </summary>
+    public interface IPersistentBlob
+    {
+        /// <summary>
+        /// Read content of a blob from storage.
```

Comment:
Consider describing the behavior/contract similar like https://github.com/open-telemetry/opentelemetry-dotnet/blob/1170f3f20d11f142f0fe171566c78170ade5d817/src/OpenTelemetry/BaseProcessor.cs#L31.

@CodeBlanch @cijothomas is the common practice to use singular form or plural form (e.g. `Read` or `Reads`)?

Reply:
@reyang This singular form works for me.

@rajkumar-rangaraj A method like `Read()` how does it know where to read from? Not `Read(Stream)`?